README.md

```diff
@@ -12,14 +12,19 @@ or
 yarn add @workos-inc/authkit-react
 ```
 
+## Setup
+
+Add your site URL to the list of allowed origins in the WorkOS dashboard by
```

Comment:
Should this be possessive or is "site URL" an intentional wording choice?

```suggestion
Add your site's URL to the list of allowed origins in the WorkOS dashboard by
```

Reply:
your way reads better :ty: